### PR TITLE
Validate form

### PR DESCRIPTION
**Pull request by Botlhale-Setou**
- Add email validation code in src files.
- Modify code for linter compliance.

### CODE REVIEW

File: scripts.js [hunk 1]
```diff
@@ -53,6 +56,22 @@ function closePopup() {
...
+
+  if (isLowerCase(mail) === false) {
+    event.preventDefault();
+    event.stopPropagation();
+    liErr.innerHTML = 'Please ensure your email is all lowercase.';
+  } else {
+    contactForm.submit();
+    liErr.innerHTML = 'Submitted successfully.';
+  }
+});
```
Comment (youmari) [inline scripts.js]: - [x] when I tried to test it and I gave it and lower case and submit the form it display the error message but not the behavior that we want it should only display the error when there is an uppercase letter so please try to fix that
- Also please try to make the error message more visible by changing its color to red and make it close to submit button

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Botlhale-Setou  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts.js (hunk 1 above)
Comment (youmari) [inline scripts.js]: - [x] also when I submit the form properly it gives me a 403 Forbidden page please try to fix that 
Screen Shot 2022-09-08 at 15 14 45

Comment (Botlhale-Setou) [reply scripts.js]: Done.

Comment (Botlhale-Setou) [reply scripts.js]: Should be fine now?

**Review by mikethreels — APPROVED** [review]
Hi Bothale, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**